modules/health-check/health-check-unary/src/main/scala/higherkindness/mu/rpc/healthcheck/service.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017-2019 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package higherkindness.mu.rpc.healthcheck
+
+import higherkindness.mu.rpc.protocol.{service, Empty, Protobuf}
+object service {
+
+  @service(Protobuf)
```

Comment:
I was thinking about providing this protocols as IDL files (.proto files), however, what we are providing here is also the implementation, so at the end, you will be adding the binary dependencies in any case. Am I right?

Reply:
Yes, but I think is better to provide it